images/versioned/binder.md

```diff
@@ -74,3 +74,9 @@ execute `/init` command with the root user specified.
 ```sh
 docker run --rm -ti -p 8787:8787 --user root rocker/binder /init
 ```
+
+## Alternatives
+
+If you are *primarily* interested in using the Jupyter ecosystem of tools, consider using
+the images from the [jupyter/docker-stacks](https://rocker-project.org/images/external.html#jupyter-docker-stacks)
```

Comment:
If I understand correctly, Quarto correctly converts relative links to HTML, so use relative links to md files here, not http.

Reply:
Like this

https://github.com/rocker-org/website/blob/f8983df9b9d1ac74507907863fda512360c14856/images/external.md?plain=1#L55-L60

Reply:
Just stumbled across this open PR. I don't have skin in the game here, but it looks like you are both close to a close and approve. @yuvipanda, a little nudge - what do you think of the requested changes?